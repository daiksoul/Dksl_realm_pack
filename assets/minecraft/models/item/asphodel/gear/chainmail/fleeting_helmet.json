{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/chainmail_helmet",
		"layer1": "misc/fleeting_symbol"
	}
}
