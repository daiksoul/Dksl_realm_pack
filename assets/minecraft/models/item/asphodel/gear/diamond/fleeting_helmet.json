{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/diamond_helmet",
		"layer1": "misc/fleeting_symbol"
	}
}
