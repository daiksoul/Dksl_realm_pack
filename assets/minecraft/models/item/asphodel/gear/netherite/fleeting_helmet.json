{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/netherite_helmet",
		"layer1": "misc/fleeting_symbol"
	}
}
