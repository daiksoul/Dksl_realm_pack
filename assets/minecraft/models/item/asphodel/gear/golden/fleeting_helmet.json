{
	"parent": "item/generated",
	"textures": {
		"layer0": "item/golden_helmet",
		"layer1": "misc/fleeting_symbol"
	}
}
